### PR TITLE
Create Add Package-Measure Energy Savings Analyses.md

### PR DESCRIPTION
**Pull request by markborkum**
#### Any background context you want to provide?

N/A

#### What does this PR do?

New proposal to add "measure in the context of a package" energy savings analyses to the schema.

#### How should this be manually tested?

No testing required.

#### What are the relevant tickets?

N/A

### CODE REVIEW

File: proposals/2022/Add Package-Measure Energy Savings Analyses.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+The term "measure-level" refers to data that are intrinsic to the measure.
+For a given `<auc:Measure>` element, measure-level energy savings analysis data are exchanged using the `<auc:MeasureSavingsAnalysis>` child element, which includes a `<auc:AnalysisSavingsByFuel>` child element.
+
+The interpretation of measure-level energy savings analysis data is that said data applies to the measure as a whole (i.e., not in the context of any specific package of measures).
+
+### Example
+
+Suppose that we have 2 arbitrary measures, #1 and #2.
+Implementation of the first measure (as a whole) would cost $100 and would provide both electricity savings only.
```
Comment (laurenliz22) [inline proposals/2022/Add Package-Measure Energy Savings Analyses.md:24]:
```suggestion
Implementation of the first measure (as a whole) would cost $100 and would provide electricity savings only.
```

**Review by laurenliz22 — APPROVED** [review]
Approve - looks great to me!  had one minor change.

**Review by nllong — APPROVED** [review]
👍

File: proposals/2022/Add Package-Measure Energy Savings Analyses.md (hunk 1 above)
Comment (markborkum) [reply proposals/2022/Add Package-Measure Energy Savings Analyses.md]: @laurenliz22 Suggested change is made.

**Review by JieXiong9119 — APPROVED** [review]
Looks good to me.

**Pull request merged**